src/radical/pilot/utils/pubsub.py

```diff
@@ -73,18 +74,15 @@ def __init__(self, flavor, channel, role, address=None):
         self._role       = role
         self._addr       = address
         self._debug      = False
-        self._logfd      = None
+        self._log        = ru.get_logger('rp.bridges')
```

Comment:
I wanted to not put this into the `radical.pilot.util` logger space, as that would be enabled by `RADICAL_PILOT_VERBOSE`, but logging on the queues can seriously impact performance.   So that would only get enabled on `RP_BRIDGES_VERBOSE=DEBUG` (which defaults, as all other levels, to `WARNING`).  It goes to stderr by default, but it probably would make sense to specify a `[self.name].log` as we do for the agent.

That RP namespace stuff might also point to the need to rethink how we handle logger hierarchies, maybe we want to do that different than suggested  by Python's logging module.  But either way, both (log target and log level spec) is for later -- the purpose of this PR is really mainly to get rid of `RADICAL_DEBUG`, w/o breaking things...